pandas/tests/plotting/test_hist_method.py

```diff
@@ -238,6 +238,20 @@ def test_hist_layout(self):
         with tm.assertRaises(ValueError):
             df.hist(layout=(-1, -1))
 
+    @slow
+    def test_tight_layout(self):
+        tm._skip_if_mpl_1()
+
+        df = DataFrame(randn(100, 2))
+        df.plot.hist()
+
+        try:
+            self.plt.tight_layout()
```

Comment:
you don't need the try/except (if it works it won't raise)

add a 1-line comment & the issue number as a comment